### PR TITLE
Add a note that SharedArrayBuffer Origin Trial is only available through http header

### PR DESCRIPTION
**Pull request by agektmr**
Updates https://developer.chrome.com/blog/enabling-shared-array-buffer/ with a note that SharedArrayBuffer Origin Trial is only available through http header

cc: @lutzvahl @jakearchibald

### CODE REVIEW

File: site/en/blog/enabling-shared-array-buffer/index.md [hunk 1]
```diff
@@ -176,12 +176,17 @@ exception expires in Chrome 96, and the exception only applies to Desktop
...
-   - If you can configure your server, you can also add the token using an
-     `Origin-Trial` HTTP header. The resulting response header should look
-     something like: `Origin-Trial: TOKEN_GOES_HERE`
+2. Add the token to your pages using an `Origin-Trial` HTTP header. The
+   resulting response header should look something like: `Origin-Trial:
+   TOKEN_GOES_HERE`
+
+{% Aside 'caution' %}
+You can only enable the origin trial using an `Origin-Trial` HTTP header. Please
+do not use a meta tag.
```
Comment (mihajlija) [inline site/en/blog/enabling-shared-array-buffer/index.md:185]:
```suggestion
You can only enable the origin trial using an `Origin-Trial` HTTP header.
Do not use a meta tag for this purpose.
```

File: site/en/blog/enabling-shared-array-buffer/index.md [hunk 2]
```diff
@@ -176,12 +176,17 @@ exception expires in Chrome 96, and the exception only applies to Desktop
...
+{% endAside %}
+
+You can verify that it's working properly by installing and testing with [Chrome
+91 beta](https://www.google.com/chrome/beta/).
```
Comment (mihajlija) [inline site/en/blog/enabling-shared-array-buffer/index.md:189]:
```suggestion
To verify that it's working properly, install [Chrome
91 beta](https://www.google.com/chrome/beta/) for testing.
```

File: site/en/blog/enabling-shared-array-buffer/index.md (hunk 1 above)
Comment (mihajlija) [reply site/en/blog/enabling-shared-array-buffer/index.md:185]: Just a note that it's best practice to be direct and avoid using "please" for instructions, it's not impolite :)

File: site/en/blog/enabling-shared-array-buffer/index.md (hunk 2 above)
Comment (mihajlija) [reply site/en/blog/enabling-shared-array-buffer/index.md:189]: Also best practice to avoid passive voice.

**Review by mihajlija — APPROVED** [review]
LGTM + comments

**Review by jakearchibald — APPROVED** [review]
LGTM. Feel free to add yourself to the authors list btw.

File: site/en/blog/enabling-shared-array-buffer/index.md [hunk 3]
```diff
@@ -176,12 +176,17 @@ exception expires in Chrome 96, and the exception only applies to Desktop
...
-   - If you can configure your server, you can also add the token using an
-     `Origin-Trial` HTTP header. The resulting response header should look
-     something like: `Origin-Trial: TOKEN_GOES_HERE`
+2. Add the token to your pages using an `Origin-Trial` HTTP header. The
+   resulting response header should look something like: `Origin-Trial:
+   TOKEN_GOES_HERE`
+
+{% Aside 'caution' %}
+You can only enable the origin trial using an `Origin-Trial` HTTP header.
+Do not use a meta tag for this purpose.
```
Comment (jpmedley) [inline site/en/blog/enabling-shared-array-buffer/index.md:185]:
```suggestion
Do not use a meta tag.
```

**Review by jpmedley — DISMISSED** [review]

**Comment by agektmr** [comment]
@rachelandrew I'd like to merge this pull request soon but it's stack with Joe's change request. Is this something you can overwrite to approve? The requested change is already fixed.

**Pull request merged**